db/seeds.rb

```diff
@@ -248,6 +248,86 @@
     fund_pledge: 1000)
 end
 
+# create patients with ALL THE INFO to play with archiving
```

Comment:
Yep, this is pretty much right on -- the only thing missing are a few call records associated with a patient, which represent our attempts to reach. You can bolt those on here with something like `patient.calls.create status: 'Reached patient', created_by: User.first` if memory serves.